py/kubeflow/kfctl/testing/pytests/kfserving_e2e_test.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+from kubernetes import client
+
+from kfserving import KFServingClient
+from kfserving import constants
+from kfserving import V1alpha2EndpointSpec
+from kfserving import V1alpha2PredictorSpec
+from kfserving import V1alpha2TensorflowSpec
+from kfserving import V1alpha2InferenceServiceSpec
+from kfserving import V1alpha2InferenceService
+from kubernetes.client import V1ResourceRequirements
+
+from ..common.utils import predict
+from ..common.utils import KFSERVING_TEST_NAMESPACE
+
+api_version = constants.KFSERVING_GROUP + '/' + constants.KFSERVING_VERSION
+KFServing = KFServingClient(config_file="~/.kube/config")
+
+
+def test_tensorflow_kfserving():
```

Comment:
Would it make more sense for this test to either live in the kubeflow/testing or kfserving repositories?

Reply:
@jinchihe I think this test is already checking out the entire kfserving repo, we can directly run the test without copying over ?

Reply:
> Would it make more sense for this test to either live in the kubeflow/testing or kfserving repositories?

Yeah, better run this in kfserving repo. I add this to address the ticket you logged: https://github.com/kubeflow/kfserving/issues/930 
Should we add some tekton tasks in `kubeflow/testing` repo, and invoke that in `kubeflow/gcp-blueprints`? Personally I suggest just check the kfserving controller state, as other repo done, agree?

Reply:
> @jinchihe I think this test is already checking out the entire kfserving repo, we can directly run the test without copying over ?

Yeah, you are right.

Reply:
See: https://github.com/kubeflow/kfserving/issues/930#issuecomment-664734634